docroot/themes/custom/yukonca_glider/yukonca_glider.theme

```diff
@@ -36,6 +40,20 @@ function yukonca_glider_preprocess_block(&$variables) {
   }
 }
 
+/**
+ * Implements hook_theme_suggestions_HOOK_alter() for page template.
+ */
+function yukonca_glider_theme_suggestions_page_alter(array &$suggestions, array $variables) {
+  // Add a template suggestions for node type.
+  /** @var \Drupal\node\NodeInterface $node */
+  if ($node = \Drupal::routeMatch()->getParameter('node')) {
+    if (is_numeric($node)) {
+      $node = Node::load($node);
+    }
+    $suggestions[] = 'page__node__' . $node->bundle();
```

Comment:
More likely we won't need this part if we will be able to combine changes with the basic twig template.

Reply:
Yes, it will be removed.